monitoring/local_monitoring/ReadMe-advancedusers.md

```diff
@@ -0,0 +1,106 @@
+# Local Monitoring with Kruize - Advanced User Guide
+
+## Getting Started
+
+To begin exploring local monitoring capabilities, follow these steps:
+
+##### Clone the demo repository:
+```sh
+git clone git@github.com:kruize/kruize-demos.git
+```
+##### Change directory to the local monitoring demo:
+```sh
+cd kruize-demos/monitoring/local_monitoring
+```
+***Note*** : We support `Kind`, `Minikube` and `Openshift` clusters.
+By default, it runs on the `Kind` cluster.
+##### Execute the demo script on kind as: 
+```sh
+./local_monitoring_demo.sh
+```
+##### Execute the demo script in openshift as: 
+```sh
+./local_monitoring_demo.sh -c openshift
+```
+
+```
+Usage: ./local_monitoring_demo.sh [-s|-t] [-c cluster-type] [-e recommendation_experiment] [-l] [-p] [-r] [-i kruize-image] [-u kruize-ui-image] [-b] [-n namespace] [-d load-duration] [-m benchmark-manifests]
+c = supports minikube, kind and openshift cluster-type
+e = supports container, namespace and gpu. Default - none.
+i = kruize image. Default - quay.io/kruize/autotune_operator:<version as in pom.xml>
+l = Run a load against the benchmark
+p = expose prometheus port
+r = restart kruize only
```

Comment:
Please update this README as well with the new options

Reply:
Done.